modules/logging-loki-cli-install.adoc

```diff
@@ -18,28 +16,242 @@ To install and configure logging on your {product-title} cluster, additional Ope
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <4>
+      type: s3 # <5>
+      credentialMode: # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify the `openshift-logging` namespace.
+<3> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<4> Specify the name of your log store secret.
+<5> Specify the corresponding storage type.
+<6> Optional field, logging 5.9 and later. Supported user configured values are as follows: `static` is the default authentication mode available for all supported object storage types using credentials stored in a Secret. `token` for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types. `token-cco` is the default value when Loki is running on managed STS mode and using CCO on STS/WIF clusters.
+<7> Specify the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
+<8> LokiStack defaults to running in multi-tenant mode, which cannot be modified. One tenant is provided for each log type: audit, infrastructure, and application logs. This enables access control for individual users and user groups to different log streams.
+
+. Apply the `LokiStack CR` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `ClusterLogging` CR object:
++
+.Example ClusterLogging CR object
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance # <1>
+  namespace: openshift-logging # <2>
+spec:
+  collection:
+    type: vector
+  logStore:
+    lokistack:
+      name: logging-loki
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+      infra:
+        maxAge: 7d
+    type: lokistack
+  visualization:
+    ocpConsole:
+      logsLimit: 15
```

Comment:
We should indicate how to increase the ocpConsole.timeout, by default 30 seconds and being hit by all the customers, it's:
```

    visualization:
      ocpConsole:
        timeout: 190s
      type: ocp-console
```

More information in https://access.redhat.com/solutions/6998442

Reply:
Also, it's needed in this section to add the "visualization.type", not present in the example